src/graph/entities/graph_entity.h

```diff
@@ -62,6 +73,12 @@ SIValue *GraphEntity_GetProperty(const GraphEntity *e, Attribute_ID attr_id);
 /* Updates existing attribute value. */
 void GraphEntity_SetProperty(const GraphEntity *e, Attribute_ID attr_id, SIValue value);
 
+/* prints the id into a buffer, returns what is the string length */
+int GraphEntity_IdToString(const GraphEntity *e, char *buffer, int bufferLen);
```

Comment:
Discard this function, it should be easy to implement as an internal static function in both node and edge,
i prefer to keep the graph_entity "api" to a bare minimum.

Reply:
done